test/integration/fdp-class.fairos.spec.ts

```diff
@@ -61,35 +59,23 @@ describe('Fair Data Protocol with FairOS-dfs', () => {
 
       await topUpFdp(fdp)
       await fdp.account.register(user.username, user.password)
-      const createdPod1 = await fdp.personalStorage.create(podName1)
+      await fdp.personalStorage.create(podName1)
       await fairos.login(user.username, user.password)
       const response = await fairos.podLs()
-      const expectedPod1 = {
-        index: 1,
-        name: podName1,
-        password: bytesToHex(createdPod1.password),
-      }
       expect(response.status).toEqual(200)
       expect(response.data).toStrictEqual({
-        pods: [expectedPod1],
-        sharedPods: [],
+        podName: [podName1],
+        sharedPodName: [],
```

Comment:
when did it become `podName` and `sharedPodName` from `pods` and `sharedPodName`?
https://github.com/fairDataSociety/FIPs/pull/61/files#diff-81f0c568292734f60520b0a790db67b5c50b148490ed1fd7ccd95283172be861R86-R89

Reply:
will change them back.